src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,17 +1,39 @@
 import React from 'react';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { setQuery, setStatus } from '../../features/filter';
+import { Status } from '../../types/Status';
+
+const statuses: Record<Status, string> = {
+  all: 'All',
+  active: 'Active',
+  completed: 'Completed',
+};
```

Comment:
```suggestion
const statuses: Record<Status, string> = {
  all: 'All',
  active: 'Active',
  completed: 'Completed',
} as const
```

Reply:
Assert this kind of objects to `const` so you will get error when you will try to mutate it.